python/py-pyobjc-cocoa/Portfile

```diff
@@ -1,52 +1,17 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           python 1.0
+PortGroup           obsolete 1.0
```

Comment:
You need to set replaced_by to something. Not doing so will give users errors when upgrading.

Reply:
OK — I seem to recall an issue where replacing a port with a previous dependency caused issues; is that still the case?

Reply:
I think you're thinking of the opposite case, where the dependency now provides a file that was previously provided by the dependent. In that case you have to use the "deactivate hack" to deactivate the dependent when upgrading the dependency.